### PR TITLE
Add synthesis / STA support for multiple liberty files.

### PR DESCRIPTION
**Pull request by mikesinouye**

### CODE REVIEW

File: static_timing/build_defs.bzl [hunk 1]
```diff
@@ -41,13 +42,14 @@ def _run_opensta_impl(ctx):
         "NETLIST": netlist.path,
         "TOP": synth_info.top_module,
         "LOGFILE": sta_log.path,
-        "LIBERTY": liberty_file.path,
-        "TCL_LIBRARY": opensta_runfiles_dir + "/tk_tcl/library",
+        "LIBERTY": default_liberty_file.path,
+        "ADDITIONAL_LIBERTIES": ",".join([f.path for f in additional_liberty_files]),
```
Comment (QuantamHD) [inline static_timing/build_defs.bzl:46]: Should we dedup with to make sure that they do not include LIBERTY?

File: static_timing/build_defs.bzl [hunk 2]
```diff
@@ -41,13 +42,14 @@ def _run_opensta_impl(ctx):
...
+        "TCL_LIBRARY": opensta_runfiles_dir + "/google3/third_party/tcl_tk/libs/library",
```
Comment (QuantamHD) [inline static_timing/build_defs.bzl:47]: This shouldn't be google3. Revert it to the old value.

File: synthesis/build_defs.bzl [hunk 3]
```diff
@@ -137,6 +140,7 @@ def _synthesize_design_impl(ctx):
         "TOP": ctx.attr.top_module,
         "OUTPUT": output_file,
         "LIBERTY": default_liberty_file,
+        "ADDITIONAL_LIBERTIES": additional_liberty_files,
```
Comment (QuantamHD) [inline synthesis/build_defs.bzl:143]: Same comment

**Review by QuantamHD — CHANGES_REQUESTED** [review]

File: static_timing/build_defs.bzl (hunk 1 above)
Comment (mikesinouye) [reply static_timing/build_defs.bzl:48]: added

File: static_timing/build_defs.bzl (hunk 2 above)
Comment (mikesinouye) [reply static_timing/build_defs.bzl]: Oops, thanks for the catch. Need to be more careful on using the patch

File: synthesis/build_defs.bzl (hunk 3 above)
Comment (mikesinouye) [reply synthesis/build_defs.bzl:145]: done

**Review by QuantamHD — APPROVED** [review]

**Pull request merged**